ncm-spma/src/test/perl/apt-list-installed.t

```diff
@@ -0,0 +1,40 @@
+# -*- mode: cperl -*-
+# ${license-info}
+# ${author-info}
+# ${build-info}
+
+=pod
+
+=head1 DESCRIPTION
+
+More tests for the C<installed_pkgs> method. This time we test that
+all the debs are printed in a format that APT will understand.
+
+=head1 TESTS
+
+These tests will run only if the dpkg binary is present.  They consist
+on retrieving the set of all installed packages and ensure there are
+no surprising strings among them.
+
+=cut
+
+use strict;
+use warnings;
+use Test::Quattor;
+use Test::More;
+use NCM::Component::spma::apt;
+
+plan skip_all => "Cannot test get_installed_pkgs, dpkg not present" if ! -x "/usr/bin/dpkg";
```

Comment:
? why is this needed? should be mocked, if not, open issue in `maven-tools` for whatever is missing.